pkg/skaffold/deploy/helm.go

```diff
@@ -105,12 +117,25 @@ func (h *HelmDeployer) deployRelease(out io.Writer, r v1alpha2.HelmRelease, b *b
 			setOpts = append(setOpts, fmt.Sprintf("%s=%s", k, v))
 		}
 	}
-
 	args = append(args, setOpts...)
-	stdout, stderr, err = util.RunCommand(exec.Command("helm", args...), nil)
+
+	execCmd := exec.Command("helm", args...)
+	stdout, stderr, err = util.RunCommand(execCmd, nil)
 	if err != nil {
 		return errors.Wrapf(err, "helm updater stdout: %s, stderr: %s", string(stdout), string(stderr))
 	}
+
+	out.Write(stdout)
+	return nil
+}
+
+func (h *HelmDeployer) deleteRelease(out io.Writer, r v1alpha2.HelmRelease) error {
+	getCmd := exec.Command("helm", h.args("delete", r.Name, "--purge")...)
+	stdout, stderr, err := util.RunCommand(getCmd, nil)
+	if err != nil {
+		logrus.Debugf("running helm delete %s: %v stdout: %s stderr: %s", r.Name, err, string(stdout), string(stderr))
```

Comment:
not necessary at all for this PR, but I've been wondering what the best way to get rid of this output boilerplate is. Maybe RunCommand can return a struct that has an `output` convenience method for printing this fmt string.